src/System.Device.Gpio/System/Device/Gpio/Drivers/RaspberryPi3Driver.Linux.cs

```diff
@@ -324,7 +324,7 @@ private void InitializeSysFS()
                 {
                     return;
                 }
-                _sysFSDriver = new UnixDriver();
+                _sysFSDriver = UnixDriver.CreateUnixDriver();
```

Comment:
For now, can you change this to always use the SysFSDriver instead? This is used strictly for eventing, which you don't support for now with libgpiod so let's make sure we force sysfs here for now.

Reply:
ok